shared/src/packages/business/libs/enums/business-api-path.enum.ts

```diff
@@ -2,6 +2,7 @@ const BusinessApiPath = {
   ROOT: '/',
   $ID: '/:id',
   DRIVERS: '/:businessId/drivers',
+  DRIVERS_BY_PAGE: '/:businessId/:pageIndex/:pageSize/drivers',
```

Comment:
do u actually need this separate route? 
I thought that for pagination we will use DRIVERS route with query params, like this
 /business/drivers?size=10&page=1